companies/exor.json

```diff
@@ -0,0 +1,19 @@
+{
+    "slug": "bitel",
+    "relevant-countries": [
+        "nl"
+    ],
+    "categories": [
+        "Holding company"
+    ],
+    "name": "Exor N.V.",
```

Comment:
The [privacy policy](https://www.exor.com/pages/disclaimer-and-privacy-policy) list various subsidiaries. Those should be listed under `runs`.